control-plane/api-gateway/gatekeeper/dataplane.go

```diff
@@ -19,7 +18,7 @@ import (
 )
 
 const (
-	allCapabilities              = "all"
+	allCapabilities              = "ALL"
```

Comment:
This was causing an odd merge when the OpenShift controller added `ALL` alongside our `all` and resulted in
```yaml
drop:
- ALL
- all
```

Reply:
Fixed: 
<img width="598" alt="image" src="https://github.com/hashicorp/consul-k8s/assets/13151671/bf1a8804-9e6c-43eb-9dec-9cbfa1135c27">